docs/01-app/05-api-reference/05-config/01-next-config-js/taint.mdx

```diff
@@ -0,0 +1,226 @@
+---
+title: taint
+description: Enable tainting Objects and Values.
+version: experimental
+---
+
+## Usage
+
+The `taint` option enables support for experimental React APIs for tainting objects and values. This feature helps prevent sensitive data from being accidentally passed to the client. When enabled, you can use:
+
+- [`experimental_taintObjectReference`](https://react.dev/reference/react/experimental_taintObjectReference) taint objects references.
+- [`experimental_taintUniqueValue`](https://react.dev/reference/react/experimental_taintUniqueValue) to taint unique values.
+
+> **Good to know**: Activating this flag also enables the React `experimental` channel for `app` directory.
+
+```ts filename="next.config.ts" switcher
+import type { NextConfig } from 'next'
+
+const nextConfig: NextConfig = {
+  experimental: {
+    ppr: 'incremental',
```

Comment:
@delbaoliveira should this be `taint: true`?

Reply:
Yeah, that was strange. Fixing up, PR is ready for review.